standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -399,19 +403,13 @@ private static ThriftServer startHttpMetastore(int port, Configuration conf)
     // Start with minWorkerThreads, expand till maxWorkerThreads and reject
     // subsequent requests
     final String threadPoolNamePrefix = "HiveMetastore-HttpHandler-Pool";
-    ExecutorService executorService = new ThreadPoolExecutor(
-        minWorkerThreads, maxWorkerThreads, 60, TimeUnit.SECONDS,
-        new SynchronousQueue<>(), new ThreadFactory() {
-      @Override
-      public Thread newThread(@NotNull Runnable r) {
-        Thread newThread = new Thread(r);
-        newThread.setName(threadPoolNamePrefix + ": Thread-" + newThread.getId());
-        return newThread;
-      }
-    });
-    ExecutorThreadPool threadPool = new ExecutorThreadPool((ThreadPoolExecutor) executorService);
+    ExecutorThreadPool executorThreadPool = new ExecutorThreadPool(maxWorkerThreads, minWorkerThreads,
```

Comment:
Could you give me some context why the change is needed? It seems netty has default idle timeout 30 seconds, but it was declared 60 seconds explicitly.